content/en/blog/becoming-a-todo-group-member-as-an-associate.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Becoming a TODO Group Member as a Linux Foundation Associate Member"
+author: todogroup
+date: 2020-05-07
+---
+
+Open source program offices have become a best practice for companies that rely on open source technologies. As open source program offices have spread, there has been increased interest in applying this best practice in non-corporate contexts. Cities, universities, and other non-corporate organizations are starting to see the benefits of creating a center of competency for open source.
+
+The TODO Group is a Linux Foundation collaborative project that serves as a working group for open source program office members around the world. In order to join the TODO Group, an organization needs to also be a member of the Linux Foundation. Many of the members of the Linux Foundations are companies or corporations who pay annual membership fees and also are paying members of one of the many sub-foundations hosted by the LF. But what do you do if you represent a muncipality or academic institution who is interested in forming an open source programs office? 
+
+Fortuntely, the Linux Foundation also offers an Associate Membership for civic, academic, and non-profit institutions who wish to join the LF and the TODO Group. Joining as an Associate Member is a simple, free process that opens up access to the TODO Group for any qualifying institution.
+
+You can view the Associate Membership application in the TODO Group's [Governance](https://github.com/todogroup/governance) repository, which also contains the TODO Charter and Membership Agreement. If you wish to join as an Associate Member, please email info@todogroup.org for more information.
```

Comment:
This is unclear. Is it an application to become an Assoc. member of LF? Or, being an LF Assoc. member already, to join TODO Group? If the former, then we should kick this to LF. It's not our place to manage their associate membership process or to host the application. If the latter then we need to be more clear about the workflow: step 1: join LF as an associate member (and link to the page/process for that); step 2: after gaining LF assoc membership, send us $this_information to $that_email. (or whatever automated process Amye mentioned the other day)

Reply:
Maybe what we need here is a short, specific call-out for the steps. "If you want to join TODO, and you work in a non-profit, academic, or civic organization: 1 - Apply for associate membership in the LF; 2- Once associate membership has been approved, email info@todogroup.org"

To clarify, the goal of this specific blog post is to clarify the path to membership in TODO for orgs that would normally only have that path through Associate Membership. I drafted the blog post to respond to inquiries about how academic or municipal institutions with budding open source programs could join TODO. The intent of the blog post isn't to document this process for the LF, it's to better serve prospective members by having something documented that we can point them to. 

Ultimately that's why I think the title "Becoming a TODO Group Member as a Linux Foundation Associate Member" is closer to the mark than "Joining the TODO Group as a Linux Foundation Associate Member" - the latter reads to me more like "If you're already a LF Associate Member, here's how to join TODO" when what I'm trying to go for is "If you want to join TODO and you're not a company, here's the path"